### ISSUE TITLE
There is no output (anti-flood) control - implement queues with configurable antiflood

### ISSUE THREAD

**Issue by Amunak** [WildPHP/Wild-IRC-Bot#23]
This is pretty important before we can go live with the rewrite, since an error or something may get the bot easily kicked for flood and potentially k-lined from the server.

- [ ] Implement basic output queue that limits the output at a certain rate
- [ ] Create multiple queues with different priority - some 'slower' and some with higher priority (you *will* want kicks and bans go through before random messages). First the messages from the highest priority queue get processed, then high priority, etc.
- [ ] Make the output rates configurable (with a safe defaults)

**Comment by Yoshi2889** [comment]
We probably want timers to be implemented for this.

**Comment by Amunak** [comment]
Yeah some kind of timing will have to be involved. I was thinking of somewhat "hard coding" it but if we make timers I guess we can just use those.

**Comment by Yoshi2889** [comment]
Should we implement the new queue manager around the say method in Bot.php?

**Comment by Yoshi2889** [comment]
We can implement queues on all outgoing data, and have a timer for the flush. How about, say, 3 seconds?

**Comment by Yoshi2889** [comment]
This can be implemented in the module that is going to handle connections, possibly? Another possibility would be to create a generic flood control module based on callbacks etc.

**Comment by Yoshi2889** [comment]
Duplicate of #65, closing.
